src/components/HealthCardDisplay/iisstandards.js

```diff
@@ -12,7 +12,7 @@ async function fetchCdcXml(file) {
   let data = await response.data;
   data = data
     .replace(/<\?xml.*\?>/g, '')
-    .replace(/&/g, '&amp;')
+    .replace(/&(?!amp;)/g, '&amp;')
     .replace(/"/g, '&quot;')
     .replace(/'/g, '&apos;');
```

Comment:
Let's do them all while we're at it
```suggestion
    .replace(/"(?!quot;)/g, '&quot;')
    .replace(/'(?!apos;)/g, '&apos;');
```

Reply:
Wouldn't this not be relevant here, since the entity in this instance wouldn't actually contain a quote or an apostrophe character?

Reply:
Ah, yup you're right.

I'm still a little wary about `.replace(/"/g, '&quot;')` since it could break XML properties outside the name, but we can handle that when preventing bad XML from being entered.